source/connect.txt

```diff
@@ -0,0 +1,367 @@
+the OCSP endpoint:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-disable-ocsp-client
+         :end-before: end-disable-ocsp-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-disable-ocsp-uri
+         :end-before: end-disable-ocsp-uri
+
+To learn more about disabling OCSP checks, see :ref:`php-disable-ocsp` in
+the TLS Configuration guide.
+
+Specify a Certificate Revocation List (CRL)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how to instruct the driver to verify the server's 
+certificate against a CRL:
+
+.. literalinclude:: /includes/usage-examples/connect-code-examples.php
+    :language: php
+    :dedent:
+    :start-after: start-crl
+    :end-before: end-crl
+
+To learn more about specifying a CRL, see :ref:`php-crl` in the TLS
+configuration guide.
+
+Present a Client Certificate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how to specify the client certificate that
+the driver presents to your MongoDB deployment:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-client-cert-client
+         :end-before: end-client-cert-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-client-cert-uri
+         :end-before: end-client-cert-uri
+
+To learn more about specifying a client certificate, see :ref:`php-client-cert` in
+the TLS Configuration guide.
+
+Provide a Certificate Key File Password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how to specify the password for your
+client certificate:
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-key-file-client
+         :end-before: end-key-file-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/usage-examples/connect-code-examples.php
+         :language: php
+         :dedent:
+         :start-after: start-key-file-uri
+         :end-before: end-key-file-uri
+
+To learn more about providing a key file password, see :ref:`php-key-file-password` in
+the TLS Configuration guide.
+
+Allow Insecure TLS
+~~~~~~~~~~~~~~~~~~
+
+The following code shows how to enable insecure TLS:
```

Comment:
Since this landing page is meant to be a quick reference for each setting, I kept the `tlsAllowInvalidCertificates` and `tlsAllowInvalidHostnames` sections (but mentioned that `tlsInsecure` is a shortcut for all three). Also added a warning admonition about the security risks